Cypress/cypress/integration/Parallel_Tests/Create_Workflow/template.spec.js

```diff
@@ -11,50 +11,22 @@ describe("Testing the workflow creation wizard using Templates", () => {
 
 	let workflowName = '';
 
```

Comment:
We will have to create an application here & also have to validate details for template workflow in the end like we are doing in other tests.

Reply:
Implemented the above changes. Thanks.